project-bom/pom.xml

```diff
@@ -31,7 +31,13 @@
             <dependency>
                 <groupId>commons-lang</groupId>
                 <artifactId>commons-lang</artifactId>
```

Comment:
I think we can remove commons-lang entirely as a direct dependency and just use commons-lang3 going forward.